tools/visinity/vis-preprocess.xml

```diff
@@ -0,0 +1,124 @@
+<tool id="visinity_preprocess" name="Create Visinity Archive" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.01">
+    <description>Create an archive of pre-processed Visinity files</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <expand macro="stdio"/>
+    <expand macro="version_cmd"/>
+    <command detect_errors="aggressive">
+        <![CDATA[
+    cp inputs.json /visinity/ &&
+    cd /visinity/ &&
+    cp "$__tool_directory__/archive_generate.py" "./archive_generate.py" &&
+    CONDAFY="micromamba run /bin/bash -c" &&
```

Comment:
What is mamba here doing? 
Please note that mamba is deprecated.

Reply:
i used a micromamba base image for easy installation while developing, as conda was visinity's recommended installation method. re: deprecation, do you have links for that? i would love to read more and wasn't able to find much with a cursory googling. the main mamba repo seems active; the closest i could find is this conversation:
- https://github.com/mamba-org/mamba/pull/3054

...where they debate the validity/utility of mamba, but seem to support continuing with micromamba development

what is your preference here? i don't super care what we use as long as it works.